web/src/main/java/uk/ac/ebi/PhenotypeArchive.java

```diff
@@ -0,0 +1,27 @@
+package uk.ac.ebi;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.support.SpringBootServletInitializer;
+import org.springframework.context.annotation.ImportResource;
+
+
+/**
+ * Created by ilinca on 24/02/2017.
+ */
+
+@SpringBootApplication
+@ImportResource("file:/Users/ilinca/IdeaProjects/PhenotypeData/web/src/main/webapp/WEB-INF/app-config.xml")
```

Comment:
Just a note for the future: We will refactor the appConfig.xml to an annotation based class next, so this won't  be needed for long